README.md

```diff
@@ -1,16 +1,47 @@
-# Pandemic Board Game (digital version)
+# Pyndemic
+_v0.1.0_
 
 ## Description
-This is an attempt to create a digital version of the original [Pandemic board game][official].
-This version is forked from [original repository][ref] and is expected to be written in Python with perspectives to become a standalone application.
+This is a future digital implementation of the original [Pandemic board game][official].
+This project is started after [this repository][ref] and is expected to be written in Python with perspectives to become a standalone application.
+
+## Project status
+The current release version is 0.1. This release is focused on implementing MVP providing minimalistic console client with limited gameplay and hotseat multiplayer introduced.
+Here's part of how a game session looks for now:
+![gameplay screenshot](/assets/readme/game_screenshot_1.png)
+
+Game mechanics:
+ - [x] Basic player actions
+ - [x] Epidemic events
+ - [x] Outbreak events
+ - [x] Win and Fail conditions
+ - [ ] Special character actions
+ - [ ] Event Cards
+ - [ ] Hand Limit
+
+## How to
+To play a game run console client in an interactive mode:
+```bash
+python3 pyndemic.py
+```
+
+For tests run:
+```bash
+python3 -m unittest
+```
+
+And for running test game session you can type:
+```bash
+python3 pyndemic.py test/test_input.txt 42
+```
 
```

Comment:
I think we need to add a section with a possible command list after all for completeness.

Reply:
Done.